V2_MIGRATION_GUIDE.md

```diff
@@ -60,164 +63,129 @@ Additionally, any classes that implemented `ParameterizableRequest` or `AuthRequ
+
+- `ParameterBuilder.GRANT_TYPE_JWT` has been removed.
+- `ParameterBuilder.ID_TOKEN_KEY` has been removed.
+- `ParameterBuilder.DEVICE_KEY` has been removed.
+- `ParameterBuilder.ACCESS_TOKEN_KEY` has been removed.
+- `ResponseType.CODE`, `ResponseType.ID_TOKEN`, and `ResponseType.ACCESS_TOKEN` have been removed.
+
+### Class and Interface changes
 
 - `SignupRequest` now implements `AuthenticationRequest` instead of the now-removed `AuthRequest`
 - `AuthorizableRequest` now extends `Request` instead of the now-removed `ParameterizableRequest`
 - `BaseCallback` has been deprecated; use `Callback` instead.
 
-### Constructors removed or changed
+### Constructor changes
 
-#### AuthenticationAPIClient
+- `AuthenticationAPIClient` can no longer be constructed from a `Context`. Use Use `AuthenticationAPIClient(client: Auth0)` instead. You can create an instance of `Auth0` using a `Context`.
+- `UsersAPIClient` can no longer be constructed from a `Context`. Use UsersAPIClient(client: Auth0, token: String)` instead. You can create an instance of `Auth0` using a `Context`.
+- `SignupRequest` now requires the second parameter to be an `AuthenticationRequest`.
+- `ProfileRequest` now requires an `AuthenticationRequest` and a `Request<UserProfile, AuthenticationException>`.
 
-- `public AuthenticationAPIClient(@NonNull Context context)`. Use `public AuthenticationAPIClient(new Auth0(context))`.
+### Methods removed or changed
 
-#### UsersAPIClient
+#### `Auth0` removals and changes
 
-- `public UsersAPIClient(@NonNull Context context, @NonNull String token)`. Use `public UsersAPIClient(new Auth0(context), token)`.
+- `setOIDCConformant(boolean enabled)` has been removed. The SDK now only supports OIDC-Conformant applications.
+- `isOIDCConformant()` has been removed. The SDK now only supports OIDC-Conformant applications.
+- `doNotSendTelemetry()` has been removed. There is no replacement.
 
-#### SignupRequest
+#### `AuthenticationAPIClient` removals and changes
 
-- `public SignUpRequest(@NonNull DatabaseConnectionRequest<DatabaseUser, AuthenticationException> signUpRequest, @NonNull AuthRequest authRequest)`. Use `public SignUpRequest(@NonNull DatabaseConnectionRequest<DatabaseUser, AuthenticationException> signUpRequest, @NonNull AuthenticationRequest authenticationRequest)` instead.
+Methods and classes specific to calling any [Authentication APIs](https://auth0.com/docs/api/authentication) categorized as Legacy have been removed in v2. The following methods have been removed:
 
-#### ProfileRequest
+- `delegation()`. Support for Legacy Authentication API endpoints have been removed.
+- `delegationWithIdToken("{ID-TOKEN}")`. Support for Legacy Authentication API endpoints have been removed.
+- `delegationWithRefreshToken("{REFRESH-TOKEN}")`. Support for Legacy Authentication API endpoints have been removed.
+- `delegationWithIdToken("{ID-TOKEN}", "{API-TYPE}}")`. Support for Legacy Authentication API endpoints have been removed.
```

Comment:
Probably worth merging all these into one line, to avoid the repeated sentence at the end.

Reply:
Updated to combine.